crates/client-api/src/routes/energy.rs

```diff
@@ -46,11 +46,10 @@ pub async fn add_energy<S: ControlStateDelegate>(
     let mut balance = ctx
         .get_energy_balance(&auth.identity)
         .map_err(log_and_500)?
-        .map(|quanta| quanta.0)
-        .unwrap_or(0);
+        .map_or(0, |quanta| quanta.get());
```

Comment:
Are the changes in this file relevant to this PR? If not, I'd prefer to see them split into a separate PR.

Reply:
Yeah, the change from .0 to .get() is part of foolproofing fuel<->quanta conversion. So that EnergyQuanta is more opaque.